ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -778,6 +781,9 @@ export class WalletDB {
     ]
 
     for (const [accountId] of await this.accountIdsToCleanup.getAll()) {
+      if (accountIdToDelete && accountIdToDelete !== accountId) {
+        continue
+      }
```

Comment:
This will work, but it looks a bit odd to have this conditional logic.

I was thinking that the logic in this loop could move to `cleanupDeletedAccount` so that the plural cleanup function calls the singular cleanup function for each account.